comments.php

```diff
@@ -25,8 +25,7 @@
 	<?php if ( have_comments() ) : ?>
 		<h2 class="comments-title">
 			<?php
-				printf( _nx( 'One thought on &ldquo;%2$s&rdquo;', '%1$s thoughts on &ldquo;%2$s&rdquo;', get_comments_number(), 'comments title', 'azera-shop' ),
-				number_format_i18n( get_comments_number() ), '<span>' . get_the_title() . '</span>' );
+				printf( _x( 'One thought on &ldquo; %s &rdquo;', 'comments title', 'azera-shop' ), get_the_title() );
```

Comment:
@claudiuvertistudio here is not ok, because not it displayes One thought .. even if there are multiple comments. it should be something like here I think https://github.com/WordPress/twentysixteen/blob/master/comments.php#L28

Also, travis is still failing. there are errors that can't be fixed?